README.md

```diff
@@ -1,13 +1,50 @@
-# Slothstagram
+## Slothstagram
```

Comment:
Since this is the top-level header, we want to use a single `#` and not a sub-heading with `##`